modules/server_new/src/clue/scala/observe/common/ObsQueriesGQL.scala

```diff
@@ -363,10 +363,7 @@ object ObsQueriesGQL:
               label
               observation {
                 label
-                program {
```

Comment:
This call is throwing an assertion on the odb but I'm unable to reproduce it in a test